core/templates/pages/exploration-editor-page/modal-templates/post-publish-modal.controller.spec.ts

```diff
@@ -64,7 +64,7 @@ describe('Post Publish Modal Controller', function() {
     var addRange = jasmine.createSpy('addRange');
     // TS ignore is used here because we are faking the getSelection function
```

Comment:
Not using TS ignore any more. Please fix.

Reply:
Please also fix all other instances of this mismatch.

Reply:
done